SimCalorimetry/HGCalSimAlgos/interface/HGCalSiNoiseMap.h

```diff
@@ -56,10 +61,10 @@ class HGCalSiNoiseMap : public HGCalRadiationMap {
   std::vector<double> &getIleakParam() { return ileakParam_; }
   std::vector<std::vector<double> > &getENCsParam() { return encsParam_; }
   std::vector<double> &getLSBPerGain() { return lsbPerGain_; }
-  std::vector<double> &getMaxADCPerGain() { return maxADCPerGain_; }
+  std::vector<double> &getMaxADCPerGain() { return chargeAtFullScaleADCPerGain_; }
 
 private:
-  //
+  //vector of three params, per sensor type: 0:120 [mum], 1:200, 2:300
```

Comment:
if we know these will always have 3 values, could use `std::array` rather than `std::vector`

Reply:
tks, addressed